ui/rdb/views/CommandsView.ts

```diff
@@ -0,0 +1,45 @@
+class CommandsView {
+    public convertArgs(inputArgs, requestArgsSchema) {
+        const args = [];
+        for (const key of inputArgs.propertyOrder) {
+            let rawArg = inputArgs.properties[key];
+            // Handles references to other definitions in schema
+            if (rawArg['$ref']) {
+                const refParts = rawArg['$ref'].split("/");
+                let rawArgRef = requestArgsSchema;
+
+                for (const key of refParts) {
+                    // Skip first entry
+                    if (key === '#') continue;
+                    rawArgRef = rawArgRef[key];
+                }
+                for (const key in rawArgRef) {
+                    rawArg[key] = rawArgRef[key];
+                }
+            }
+            args.push({
+                ...rawArg,
+                id: key
+            });
+        }
+        return args;
+    }
+
+    public processArgToSendToExtension(argType: string, argValue: string) {
+        if (argType == 'boolean') {
+            return argValue === 'true'
+        } else if (argType == 'array' || argType == 'object') {
+            return JSON.parse(argValue);
+        } else if (argType == 'number') {
+            return Number(argValue);
```

Comment:
We probably don't the the object/boxed version of a number here. Perhaps consider just using `parseFloat` instead?

https://stackoverflow.com/questions/11988547/what-is-the-difference-between-number-and-parsefloat

Reply:
In that link you provided it mentions 
```
When Number is called as a function rather than as a constructor, it performs a type conversion...

Returns a Number value (not a Number object) computed by ToNumber(value) if value was supplied, else returns +0.
```
If I did `return new Number(argValue);` that would return a Number object else it returns a `number` type whether I do it the current way or `parseFloat`. Happy to change it as I realize all numbers in Javascript are floats under the hood. The Number() syntax just seemed to match better with the argType.